source/appendices/e-audit-requirements-and-tests/02-audit-tests.rst

```diff
@@ -5,1045 +5,391 @@
 
-**IMPORTANT NOTE**: This test is required when investigations can be reported independently of a request. It is not required when all investigations have a matching request. Any exclusion of tests must be confirmed by COUNTER prior to testing and the auditor notified.
+In order for the Item Report to be accurately audited, the report provider MUST supply the auditor with a list of Data_Types represented on the platform.
 
-The auditor will perform audit tests based on the relevant options detailed below.
 
-The audit test consists of clicking links to an investigation of an item within a book twice in succession (double-clicks). If the two clicks occur within a 30-second time-span, only the second Total_Item_Investigations must be recorded. If the two clicks occur with more than 30 seconds between, then 2 Total_Item_Investigations must be counted. In both cases only 1 Unique_Item_Investigations and 1 Unique_Title_Investigations will be reported.
+E.2.6.1 Total_Item_Requests and Unique_Item_Requests
+''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-**Option 1**: Content provider offers OA_Gold items in addition to Controlled.
+The auditor MUST make a total of 100 requests on 50 unique items representing the scope of the platform. That is, where a platform is made up of a mixture of content with Data_Types Article, Multimedia and Patent, the auditor should represent each of those Data_Types proportionately in the audit test.
 
-The auditor must carry out a total of 32 tests. Each test will consist of 2 item investigations. There are 2 types of tests that must be carried out:
+This MUST result in 100 Total_Item_Requests and 50 Unique_Item_Requests. The auditor MUST record the Data_Types for each item (e.g. Multimedia) and the resulting Item Report MUST reflect those records.
 
-* “Inside” tests (Two investigations are made to the same book item, and the second investigation is made within 30 seconds of the first).
-* “Outside” tests (Two investigations are made to the same book item, and the second investigation is made more than 30 seconds after the first).
 
-The auditor must carry out 16 **inside** tests. This requires 8 Investigations to book items where the Access_Type is Controlled and 8 investigations to book items where the Access_Type is OA_Gold.
+E.2.7 Audit Tests for Double-Click Filtering
+""""""""""""""""""""""""""""""""""""""""""""
 
-Each book must have 2 item tests. This will report 2 Total_Item_Investigations, 2 Unique_Item_Investigations, and 1 Unique_Title_Investigations.
+This audit test applies to investigations and requests metrics across all COUNTER Reports and should represent the scope of the platform. That is, where a platform is made up of a mixture of content with Data_Types Article, Multimedia and Patent, the auditor should represent each of those Data_Types proportionately in the audit test.
```

Comment:
I would suggest to use a different example than in the previous section for the Item Report tests.